spec/placeholder.spec.js

```diff
@@ -66,14 +66,24 @@ describe('Placeholder TestCase', function () {
     it('should add the default placeholder text when data-placeholder is not present', function () {
         var editor = new MediumEditor('.editor'),
             placeholder = window.getComputedStyle(editor.elements[0], ':after').getPropertyValue('content');
-        expect(placeholder).toEqual("'" + editor.options.placeholder + "'");
+        if (placeholder.indexOf('attr(') === 0) {
+            // In firefox, getComputedStyle().getPropertyValue('content') can return attr() instead of what attr() evaluates to
+            expect(placeholder).toEqual('attr(data-placeholder)');
+        } else {
```

Comment:
to reduce duplication, you _could_ extract this `if/else` into a function here in placeholder.spec.js that accepts `placeholder` as an argument.